tests/shutdown/svirt_upload_assets.pm

```diff
@@ -38,6 +39,8 @@ sub extract_assets {
 }
 
 sub run {
+    # Skip if the os-autoinst backend implements this feature
+    return 1 if defined &backend::svirt::do_extract_assets;
```

Comment:
I assume this should stay **as-is** and **not** follow the `can` approach from #1895, because this is specifically about wether _svirt_ implements the method.

Reply:
#1895 ?

Yes.

Reply:
Sorry, linking between repos on GitHub sucks. Of course I meant https://github.com/os-autoinst/os-autoinst/pull/1895#pullrequestreview-861472088